### DISCUSSION TITLE
Does ScrollViewer need to set MaxHeight or Height in order to work properly?

### DISCUSSION THREAD

**Discussion by IronEaterBeast** [AvaloniaUI/Avalonia#15268] [Q&A]
I placed ScrollViewer in a grid, but the grid's height is subject to change based on the size of the entire window. Therefore, I cannot give ScrollViewer an exact height. In this case, how can I use it?

`<Grid RowDefinitions="Auto,Auto" Grid.Column="1">
<ScrollViewer Grid.Row="0" MaxHeight="700" AllowAutoHide="False">`

Or, is there any other way to use scroll in Grid?

**Comment by timunie** [comment]
```diff
<Grid 
!     RowDefinitions="*,Auto" 
      Grid.Column="1"> 
    <ScrollViewer Grid.Row="0" MaxHeight="700" AllowAutoHide="False">
... 
```

More: https://docs.avaloniaui.net/docs/reference/controls/grid/#proportional-size-definitions
